### DISCUSSION TITLE
Cloudflare Pages build fails

### DISCUSSION THREAD

**Discussion by noticef** [mearashadowfax/ScrewFast#83] [General]
I cannot seem to deploy using Cloudflare Pages. I tried a fresh repository, also checked the [outDir](https://docs.astro.build/en/reference/configuration-reference/#outdir) setting and also tried installing [@astrojs/cloudflare](https://docs.astro.build/en/guides/integrations-guide/cloudflare/) all with no joy. Any help is greatly appreciated as this template is very well designed.
![Screenshot 2024-05-15 at 9 07 12 AM](https://github.com/mearashadowfax/ScrewFast/assets/159450260/82185f9f-dcb9-49e7-a274-533237210a3b)

**Comment by mearashadowfax** [comment]
Hello @noticef! I tested deploying an Astro site to Cloudflare Pages and it worked after making a small change. In your `astro.config.mjs` file, you can comment out or remove the lines:  `import vercelStatic from "@astrojs/vercel/static"` and `adapter: vercelStatic()`. As per Astro's docs, if you're using Astro as a static site builder, you don't need an adapter for Pages.

When setting up your project in Cloudflare Pages, simply select `Astro` from the list of frameworks and leave the other settings as default.
This should resolve the deployment issue. 

![image](https://github.com/mearashadowfax/ScrewFast/assets/125820963/6b52d5c2-7534-4748-9660-f2d325bc19d9)

**Reply by noticef** [reply]
THANK YOU @mearashadowfax ! 
I had already selected Astro from the list of frameworks, but after I commented out the line:  adapter: vercelStatic().
It was SUCCESS. Again thank you for your share and the help. Keep up the good work.
